applications_tests/lethe-fluid/cahn_hilliard_navier_stokes_energy_postprocessing.prm

```diff
@@ -0,0 +1,127 @@
+set dimension = 2
+
```

Comment:
We don't add the subsection comments anymore? (Like, the ***************)

Reply:
Agreed but some tests have it and some other don't, I would say that the test's .prm must be as minimalists as possible, hence the absence of subsection decorations. Can be discussed though